lib/build.js

```diff
@@ -270,7 +270,12 @@ class BuildModel extends BaseModel {
             }
 
             return Promise.all(updateTasks);
-        });
+        })
+            .catch((err) => {
+                winston.error(`Failed to update commit status: ${err}`);
+
+                return Promise.resolve();
```

Comment:
if the caller does not care about return values, this should be good. If all pass it returns an array, in this case nothing.

Reply:
yeah, i checked where it calls the function, it doesn't use the results resolve by this